bsk_rl/utilities/gymnasium/initial_conditions/leo_initial_conditions.py

```diff
@@ -0,0 +1,324 @@
+import numpy as np
+from Basilisk.utilities import astroFunctions
+from Basilisk.utilities import macros as mc
+from Basilisk.utilities import orbitalMotion
+from basilisk_env.sim_utilities.initial_conditions import leo_orbit, sc_attitudes
```

Comment:
should be bsk_rl import

Reply:
On a related note, I don't think we talked about this but the `gymnasium` layer seems unnecessary in utilities based on our convo about minimizing folder depth; could just be `utilities/effector_primatives` and `utilities/initial_conditions`

Reply:
Agreed and done!